_data/compatibility/blockchaininfo.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: blockchain.info
+internal_url: /en/compatibility/blockchaininfo
+logo: /img/compatibility/blockchaininfo/blockchaininfo.png
+rbf:
+  tested:
+    date: "2019-07-05"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "false"
+      list: "true"
+      details: "false"
+      shows_replaced_version: "true"
```

Comment:
This gets a checkmark for "Shows replaced transaction only".  Is that your intention?  (I was under the impression that you want the interface to show both the original and the replaced version when possible.)

Reply:
@harding it’s a bit subjective from a usability perspective. I can see only showing replacement being good because it is simpler for less experienced users, but I can see both showing being "better" for more experienced users. what do you think?

Reply:
I personally think wallets should show only show one version of transaction, particularly the (1) worst-case (2) most-likely transaction.  E.g.,

1. Worst case: of all conflicting unconfirmed transactions, only display a version that pays the user the least amount
2. Most-likely: if there are multiple worst-case transactions all paying the user the same amount, only display the one with the highest feerate.

However, giving a checkmark to wallets only showing the latest (e.g. highest feerate) replacement seems fine to me.

Reply:
Good points. Didn’t touch either of those scenarios with existing testing. 

> However, giving a checkmark to wallets only showing the latest (e.g. highest feerate) replacement seems fine to me.

We will leave as we have it for now then?